src/torrent.cpp

```diff
@@ -7044,9 +7044,7 @@ namespace libtorrent
 		if (m_completed_time != 0 && m_completed_time < m_added_time)
 			m_completed_time = m_added_time;
 
-		// load file priorities except if the add_torrent_param file was set to
-		// override resume data
-		if (!m_override_resume_data || m_file_priority.empty())
+		if (m_file_priority.empty())
```

Comment:
hm.. I need a test for this. It's not obvious this is correct. This looks like the priorities in add_torrent_params always take priority.